### DISCUSSION TITLE
Increase volume slider length

### DISCUSSION THREAD

**Discussion by TemplarB** [jeffvli/feishin#363] [Ideas]
I usually send audio on my amplifier that has the volume set to max and I control volume from the program. However the volume slide is rather short, so it is hard to fine-tune the volume

**Comment by jeffvli** [comment]
You can use your mouse scroll wheel to do fine adjustments to the volume.

There is also a setting under `General` called `Volume wheel step` which adjusts the amount the scroll wheel changes the volume.
